lib/AST/GenerateAST.cpp

```diff
@@ -502,6 +589,7 @@ GenerateAST::Result GenerateAST::run(llvm::Function &func,
   // structurization
   llvm::ReversePostOrderTraversal<llvm::Function *> rpo(&func);
   rpo_walk.assign(rpo.begin(), rpo.end());
+  GetOrCreateReachingCond(rpo_walk[0]);
```

Comment:
Walking all the blocks in `rpo_walk` causes the condition explosion. Only inspecting the first block seems to work, but I cannot say I'm sure it's correct in general.